packages/account-v2/src/constants/errorMessages.tsx

```diff
@@ -1,4 +1,6 @@
-export const API_ERROR_CODES = {
+type TErrorConfig = Readonly<Record<string, Readonly<{ code: string | null; message: string }>>>;
```

Comment:
should we use here just string for types?
I think TS will not work with proper suggestion if  we use just IDV_ERROR_CODES object.
would it be better to mark config object as const?